pages/single-form.jsx

```diff
@@ -91,6 +91,7 @@ var SingleForm = React.createClass({
         </div>
         <SmallPrint stripeNotice={true} />
         <Footer/>
+        <script async src="https://checkout.stripe.com/checkout.js"></script>
```

Comment:
The user would need to select an amount, click the privacy checkbox, and then the donate checkbox. If you're worried about that case we could add an existence guard to the StripeCheckout.configure() block